### DISCUSSION TITLE
bigfix/11.0/platform/Platform/Config/c_filldb_buffer_directory

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#47] [Q&A]
# bigfix/11.0/platform/Platform/Config/c_filldb_buffer_directory

The FillDB buffer directory temporarily stores reports from the clients before they are stored into the database.

https://help.hcltechsw.com/bigfix/11.0/platform/Platform/Config/c_filldb_buffer_directory.html?hl=filldb

**Comment by JoeBrunsTR** [comment]
For the 2 windows settings, we MUST have the registry key type.  Is it DWORD, is it STRING?  Common guys.
